Storage/tests/System/IamConfigurationTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Storage\Tests\System;
+
+use Google\Cloud\Storage\StorageObject;
+use GuzzleHttp\Client;
+
+/**
+ * @group storage
+ * @group storage-iamconfiguration
+ */
+class IamConfigurationTest extends StorageTestCase
+{
+    private $guzzle;
+
+    public function setUp()
+    {
+        $this->guzzle = new Client;
+    }
+
+    public function testBucketPolicyOnly()
+    {
+        $bucket = self::createBucket(self::$client, uniqid(self::TESTING_PREFIX));
+
+        $bucket->update($this->bucketConfig());
+
+        $info = $bucket->reload();
```

Comment:
We should be able to drop these `reload` calls, as `update` returns the adjusted metadata as well.

Reply:
Thanks! All set. :)